### DISCUSSION TITLE
Is there a command to get screen size?

### DISCUSSION THREAD

**Discussion by mega6453** [doronz88/pymobiledevice3#1252] [Q&A]
I want to get the screen size. So, is there any command that gives the screen size?

**Comment by doronz88** [comment]
Create a new subcommand for that.
You can check it out here:
https://github.com/doronz88/pymobiledevice3/pull/1254

Unfortunately, `fastapi` is currently partially with bugs, so we'll have to wait before re-running the CI and releasing

**Reply by mega6453** [reply]
looks like this is merged.. which command should we use to get screen size? Thanks

**Reply by doronz88** [reply]
```shell
pymobiledevice3 springboard homescreen-icon-metrics
```

**Reply by Art-S-D** [reply]
Is there a way to get it in pixels rather than points?
I searched for the pixel ratio of the device but I couldn't find it

**Reply by doronz88** [reply]
This was already answered in the discord channel by @AnNEDoMini :

```none
pymobiledevice3 lockdown get com.apple.mobile.iTunes ScreenWidth
1290

pymobiledevice3 lockdown get com.apple.mobile.iTunes ScreenHeight
2796

pymobiledevice3 lockdown get com.apple.mobile.iTunes ScreenScaleFactor
3.0
```
